requirements-dev-frozen.txt

```diff
@@ -166,6 +166,7 @@ selenium==4.3.0
 sentry-arroyo==2.9.1
 sentry-cli==2.16.0
 sentry-kafka-schemas==0.0.29
+sentry-redis-tools==0.1.4
```

Comment:
we should bump these all to 1.x.x -- if we're using it in production they shouldn't be cowards-versioned